test_defopt.py

```diff
@@ -24,6 +25,18 @@
     unittest.TestCase.assertRaisesRegex = unittest.TestCase.assertRaisesRegexp
 
 
+class Choice(Enum):
+    one = 1
+    two = 2
+
+
+if hasattr(typing, 'NamedTuple'):
```

Comment:
When is this not the case? It looks like `typing.NamedTuple` has been present since its introduction in Python 3.5.